Protocol sync updates (#394)
MukurtuCMS/mukurtu-user-manual: docs/media/ByTypeMediaUpload/Audio.md

    ![Screenshot of the file and filename showing how it automatically filled in.](../../_embeds/audiofiles2.png)

> 5. Use the toggle(s) to apply cultural protocols to the media asset. Media created within a content form will automatically inherit content protocols. 

Reviewer: ```suggestion
5. Use the toggle(s) to apply cultural protocols to the media asset. By default, media assets created within a content form will sync and inherit protocols from that content. 
```

    !!! tip
        All is the default setting.

> 7. Optionally, you may select the **Sync protocols with content** setting. When enabled, this setting ensures the media asset's protocols will always match those of it's parent content to keep content and media access in sync. Media created within a content form will automatically inherit content protocols. Media created elsewhere require protocol selection, but will sync once added to content. You can disable the sync setting to set media protocols independent of parent content. 

Reviewer: ```suggestion
7. When enabled, the **Sync protocols with content** setting ensures the media asset's protocols will always match those of it's parent content to keep content and media access in sync. Media with synced protocols created within a content form will sync and inherit protocols from that content. Media created elsewhere require protocol selection, but will sync once added to content. You can disable the sync setting to set media protocols independent of parent content. 
```

Reviewer: Make these same changes on all the other specific media documents here.
